src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,48 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './CurrentUser.scss';
+import { findUser } from '../../api/api';
+import { User } from '../../react-app-env';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  userId: number;
+  deleteUser: () => void;
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<Props> = React.memo(({
+  userId,
+  deleteUser,
+}) => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
```

Comment:
If you want, you could write simply:
`const [selectedUser, setSelectedUser] = useState<User>();`

Reply:
better with null